clients/nimbus-bn/nimbus_bn.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+if [ ! -f "/hive/input/genesis.ssz" ]; then
+    if [ -z "$HIVE_ETH2_ETH1_RPC_ADDRS" ]; then
+      echo "genesis.ssz file is missing, and no Eth1 RPC addr was provided for building genesis from scratch."
+      # TODO: alternative to start from weak-subjectivity-state
+      exit 1
+    fi
+fi
+
+mkdir -p /data/testnet_setup
+
+cp /hive/input/genesis.ssz /data/testnet_setup
+cp /hive/input/config.yaml /data/testnet_setup
+
+# znrt encodes the values of these items between double quotes (""), which is against the spec:
+# https://github.com/ethereum/consensus-specs/blob/v1.1.10/configs/mainnet.yaml
+# Nimbus is the only client that complains about this.
+sed -i 's/TERMINAL_TOTAL_DIFFICULTY: "\([[:digit:]]\+\)"/TERMINAL_TOTAL_DIFFICULTY: \1/' /data/testnet_setup/config.yaml
+sed -i 's/TERMINAL_BLOCK_HASH: "\(0x[[:xdigit:]]\+\)"/TERMINAL_BLOCK_HASH: \1/' /data/testnet_setup/config.yaml
+sed -i 's/GENESIS_FORK_VERSION: "\(0x[[:xdigit:]]\+\)"/GENESIS_FORK_VERSION: \1/' /data/testnet_setup/config.yaml
+sed -i 's/ALTAIR_FORK_VERSION: "\(0x[[:xdigit:]]\+\)"/ALTAIR_FORK_VERSION: \1/' /data/testnet_setup/config.yaml
+sed -i 's/BELLATRIX_FORK_VERSION: "\(0x[[:xdigit:]]\+\)"/BELLATRIX_FORK_VERSION: \1/' /data/testnet_setup/config.yaml
+sed -i 's/SHARDING_FORK_VERSION: "\(0x[[:xdigit:]]\+\)"/SHARDING_FORK_VERSION: \1/' /data/testnet_setup/config.yaml
+echo Using config.yaml
+cat /data/testnet_setup/config.yaml
+echo ""
+
+echo "${HIVE_ETH2_CONFIG_DEPOSIT_CONTRACT_ADDRESS:-0x1111111111111111111111111111111111111111}" > /data/testnet_setup/deposit_contract.txt
+echo "${HIVE_ETH2_DEPOSIT_DEPLOY_BLOCK_NUMBER:-0}" > /data/testnet_setup/deploy_block.txt
+
+mkdir -p /data/beacon
+chmod 0700 /data/beacon
+
+LOG=INFO
```

Comment:
Not sure if people are expected to look at the log output in any meaningful way here, but if the disk resources are available, it's useful to have debug logs.

Reply:
The log level can be configured by hive, see below.